theseus/theseus_layer.py

```diff
@@ -45,6 +46,39 @@ def forward(
         )
         return values, info
 
+    def compute_samples(
+        self,
+        linear_solver: LinearSolver = None,
+        n_samples: int = 10,
+        temperature: float = 1.0,
+    ) -> torch.Tensor:
+        if linear_solver is None:
+            return None
```

Comment:
raise error?

Reply:
Returned None here since if learning_method is set to "leo" and there isn't a valid sampling function, this would revert to a default MMP algorithm where it uses the mean trajectory. I can change it to a RaiseError if that seems better.

Reply:
Actually this is better. We should just document this in the comments.

Reply:
Sounds good, documented in the comments.